install/sensors.csv

```diff
@@ -949,6 +949,7 @@ Nanometrics,Trillium Compact 120PH-2,9336,PAKZ,10,0,,0,1.2,0,0,0,0,2022-12-19T00
 Nanometrics,Trillium Compact 120PH-2,9337,PTAZ,10,0,,0,1.2,0,0,0,0,2022-12-08T21:30:00Z,9999-01-01T00:00:00Z
 Nanometrics,Trillium Compact 120PH-2,9365,HTPZ,10,0,,0,1.2,0,0,0,0,2022-12-19T02:31:00Z,9999-01-01T00:00:00Z
 Nanometrics,Trillium Compact 120PH-2,9366,WTAZ,13,138,,0,90,0,0,0,0,2023-06-20T00:00:00Z,2025-03-10T20:42:00Z
+Nanometrics,Trillium Compact 120PH-2,9366,MBAZ,10,0,,0,93,0,0,0,0,2025-03-12T01:44:00Z,9999-01-01T00:00:00Z
```

Comment:
Kia Ora @Thomas-Benson @gva6500 @CBurton90 
Would you confirm the sensor Depth  since it is the same as the previous ?

Reply:
Can post hole going that deep?

Reply:
Can go up to 300 m, assuming normal temperatures